### ISSUE TITLE
Always showing a noise images

### ISSUE THREAD

**Issue by AI-Ahmed** [lucidrains/imagen-pytorch#236]
Hello, @lucidrains I hope you are doing well. I have reserved 10 GPUs of a server for doing distribution training with `imagen-pytorch`, I trained it on a 40k dataset of **LAION-Dataset** text-to-image methodology. I trained it for 9 hours and what I got it that;

![image](https://user-images.githubusercontent.com/72295771/194351828-d4e5d66b-d431-436c-ab52-7ab5cf8f1853.png)

The images normalized between -1, 1. with 2 Unets.. would you mind if you told us the reasons that can give us this kind of output, If you would like to take a look at the code, we would love it, too.

We would really appreciate it as the community to know the reasons that would give these kinds of results (I and the team had followed up with all the instructions that have been made here).

Thanks,

**Comment by lucalevi** [comment]
@AI-Ahmed Thanks for the question, following it.

How did you train Imagen on your dataset, i.e. what kind of commands did you use?

As far as I understand, one has to adapt the following two lines to one's own needs - with proper text and images
`text_embeds = torch.randn(4, 256, 768).cuda()
images = torch.randn(4, 3, 256, 256).cuda()`

The question is: how? Can you share your code? Thanks!

**Comment by AI-Ahmed** [comment]
Hello @lucalevi,
   Thank you so much for answering my request. Ans you can see in the first screenshot regards the text embedding, we are using T5 to tokenize the text and embed it into `cuda` device:
   
![image](https://user-images.githubusercontent.com/72295771/194769695-38c245bb-2df8-482b-985e-442f5f849d23.png)

In terms of the image, we `transform` it and then normalize it between `(-1, 1)`.

![image](https://user-images.githubusercontent.com/72295771/194769743-6ca6aede-61d3-4609-8a0d-d8b8f9047b94.png)
